### DISCUSSION TITLE
Netherlands: EnergyZero

### DISCUSSION THREAD

**Discussion by Damazinho** [JaccoR/hass-entso-e#47] [Price Modifyer Templates]
{% set s = {
"extra_cost": 0.013504587,
"energie_belasting": 0.036788991,
"ode_belasting": 0.030504587
} 
%} 
{{(current_price + s.extra_cost + s.energie_belasting + s.ode_belasting) | float}}

**Comment by maartenlandman89** [comment]
Thanks! I’m waiting for the API and than I will try this. When it’s works, I will let you know 😀

**Reply by Damazinho** [reply]
It works, believe me

**Reply by maartenlandman89** [reply]
Oh for sure! It’s not the code, it’s more my knowledge level 😋

**Reply by maartenlandman89** [reply]
Damazinho, I installed the addon but the prices are not the same as on EnergyZero. Can you please help me? This are the settings: 
![1C978BC9-C5F4-4A5D-8B5D-3DA61D2778AD](https://user-images.githubusercontent.com/78348610/203520838-048b5453-56e8-4830-81b3-366fdb2e20ce.jpeg)

**Reply by Damazinho** [reply]
> Damazinho, I installed the addon but the prices are not the same as on EnergyZero. Can you please help me? This are the settings: ![1C978BC9-C5F4-4A5D-8B5D-3DA61D2778AD](https://user-images.githubusercontent.com/78348610/203520838-048b5453-56e8-4830-81b3-366fdb2e20ce.jpeg)

that's because they shouldn't be the same. EnergyZero desplays prices only with VAT (BTW), but the price when using this template is the All-in price, so with leveringskosten, energiebelasting etc.

**Reply by maartenlandman89** [reply]
Thank you for your reply. I get that.. But when I add a new addon without the extra cost, only with Tex, the price is still different. Maybe it’s juist me.. 😅 If you set this is correct, I’m ok with it 😊

**Reply by maartenlandman89** [reply]
Damazinho, did you find a way to import the gas prices of EnergyZero to home assistant? This way I can make a correct total cost of energy.

**Reply by Damazinho** [reply]
> Damazinho, did you find a way to import the gas prices of EnergyZero to home assistant? This way I can make a correct total cost of energy.

yes, see also: https://github.com/JaccoR/hass-entso-e/discussions/35#discussioncomment-3941379

**Reply by maartenlandman89** [reply]
Perfect! Thank you 😀

**Comment by devblaz** [comment]
Energy taxes have been changed in 2023. Also, EnergyZero has changed their extra costs.
I think this should be the current script:

{% set s = {
"extra_cost": 0.01750,
"energie_belasting": 0.12599,
"ode_belasting": 0.0
}
%}
{{(current_price + s.extra_cost + s.energie_belasting + s.ode_belasting) | float}}

**Comment by olivier-van-wijngaarden** [comment]
GreenChoice uses EneryZero as their sub-provider for energy with dynamic prices. Are the costs the same?

**Reply by devblaz** [reply]
I guess so. I use Energie VanOns as supplier and they also use EnergyZero and the price is the same.
